features/steps/step_checking_status.py

```diff
@@ -0,0 +1,61 @@
+from behave import when, then
+from selenium import webdriver
+from selenium.webdriver.support.wait import WebDriverWait
+from helpers.icon_in_div_changed import IconInDivChanged
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+
+
+FULL_NAME = None
+driver = webdriver.Chrome
+status_shortcuts = {'available': 'icon-avail',
+                    'away': 'icon-xa',
+                    'do not disturb': 'icon-dnd'}
+
+
+def change_FN(name):
+    global FULL_NAME
+    FULL_NAME = name
+
+
+@when('we get full user name from settings')
+def get_full_name(context):
+    global FULL_NAME
+    FULL_NAME = context.settings_page.full_name()
```

Comment:
"context.hipchat_full_name = context.settings_page.full_name()" ?

Reply:
Thanks. Global variable FULL_NAME is changed to context.hipchat_full_name now. But instead of doing the initialization of this variable in before_all(context) function it was done in @when('we get full user name from settings') step because we need to be logged in.